2.process-data/scripts/nbconverted/1.process-ebimage-features.py

```diff
@@ -2,18 +2,18 @@
 # coding: utf-8
 
 # # Process EBImage-based Features
-# 
+#
```

Comment:
Same comment about autogenerated files - lets remove the commits with a [`rebase`](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request) or commit the files after running:

```bash
jupyter nbconvert --to=script \
  --FilesWriter.build_directory=scripts/nbconverted \
  *.ipynb
```